easybuild/easyconfigs/r/R/R-4.1.0-foss-2021a.eb

```diff
@@ -0,0 +1,2926 @@
+name = 'R'
+version = '4.1.0'
+
+homepage = 'https://www.r-project.org/'
+description = """R is a free software environment for statistical computing
+ and graphics."""
+
+toolchain = {'name': 'foss', 'version': '2021a'}
+
+source_urls = ['https://cloud.r-project.org/src/base/R-%(version_major)s']
+sources = [SOURCE_TAR_GZ]
+patches = ['%(name)s-%(version)s_identify-flexiblas-in-configure.patch']
+checksums = [
+    'e8e68959d7282ca147360fc9644ada9bd161bab781bab14d33b8999a95182781',  # R-4.1.0.tar.gz
+    '2c6720e2e144ae4fe00842daab0ebba72241080603e0ff1a6ca758738041b257',  # R-4.1.0_identify-flexiblas-in-configure.patch
+]
+
+builddependencies = [
+    ('pkg-config', '0.29.2'),
+    ('Xvfb', '1.20.11'),
+]
+dependencies = [
+    ('X11', '20210518'),
+    ('Mesa', '21.1.1'),
+    ('libGLU', '9.0.1'),
+    ('cairo', '1.16.0'),
+    ('libreadline', '8.1'),
+    ('ncurses', '6.2'),
+    ('bzip2', '1.0.8'),
+    ('XZ', '5.2.5'),
+    ('zlib', '1.2.11'),
+    ('SQLite', '3.35.4'),
+    ('PCRE2', '10.36'),
+    ('libpng', '1.6.37'),  # for plotting in R
+    ('libjpeg-turbo', '2.0.6'),  # for plottting in R
+    ('LibTIFF', '4.2.0'),
+    ('Java', '11', '', True),
+    ('Tk', '8.6.11'),  # for tcltk
+    ('cURL', '7.76.0'),  # for RCurl
+    ('libxml2', '2.9.10'),  # for XML
+    ('GMP', '6.2.1'),  # for igraph
+    ('NLopt', '2.7.0'),  # for nloptr
+    ('FFTW', '3.3.9'),  # for fftw
+    ('libsndfile', '1.0.31'),  # for seewave
+    ('ICU', '69.1'),  # for rJava & gdsfmt
+    ('HDF5', '1.10.7'),  # for hdf5r
+    ('UDUNITS', '2.2.28'),  # for units
+    ('GSL', '2.7'),  # for RcppGSL
+    ('ImageMagick', '7.0.11-14'),  # for animation
+    ('GLPK', '5.0'),  # for Rglpk
+    ('nodejs', '14.17.0'),  # for V8 (required by rstan)
+    ('GDAL', '3.3.0'),  # for sf
+    ('git', '2.32.0', '-nodocs'),
```

Comment:
@branfosj This should be `libgit2` rather than `git`...

```python
    ('libgit2', '1.1.0'),

Reply:
1720ad2